JavaPrograms/src/main/java/Calculator.java

```diff
@@ -0,0 +1,46 @@
+import java.util.Scanner;
+
+/**
+ * Created by webonise on 14/9/16.
+ */
+public class Calculator {
```

Comment:
This is not as per expectation. Calculator class should only contain methods like add, subtract etc. And create simple demo class which will access calculator object. Change the whole implementation.